### PR TITLE
fix[lang]: define rounding mode for sqrt

### PR DESCRIPTION
**Pull request by charles-cooper**
### What I did
fix for https://github.com/vyperlang/vyper/security/advisories/GHSA-2p94-8669-xg86

prior to this PR, the rounding mode for `sqrt()` is undefined. this PR ensures the result is rounded down.

### Commit message

```
prior to this commit, the rounding mode for `sqrt()` is undefined,
which could be an issue for applications which use `sqrt()` to
determine boundary conditions. this commit ensures the result is
rounded down.
```

### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: tests/functional/codegen/types/numbers/test_sqrt.py [hunk 1]
```diff
@@ -146,6 +146,9 @@ def test_sqrt_bounds(sqrt_contract, value):
 )
 @hypothesis.example(value=Decimal(SizeLimits.MAX_INT128))
 @hypothesis.example(value=Decimal(0))
+@hypothesis.example(value=Decimal("0.9999999998"))
```
Comment (cyberthirst) [inline tests/functional/codegen/types/numbers/test_sqrt.py:149]: please leave a link to the advisory to explain the values https://github.com/vyperlang/vyper/security/advisories/GHSA-2p94-8669-xg86

**Comment by cyberthirst** [comment]
let's please also mention the rounding behavior in the documentation

File: tests/functional/codegen/types/numbers/test_sqrt.py (hunk 1 above)
Comment (charles-cooper) [reply tests/functional/codegen/types/numbers/test_sqrt.py:150]: a04254e143e7782c7d5eebdb7b764bb88ebdfa11

**Comment by charles-cooper** [comment]
> let's please also mention the rounding behavior in the documentation

8157da8ac9c8f28858809366f6838fca66a5308d

**Review by cyberthirst — APPROVED** [review]
